### PR TITLE
Use ocf:heartbeat:clvm instead of ocf:lvm2:clvmd

### PR DESCRIPTION
**Pull request by dzyuzin**
Since sle12-sp2 we recommend to use clvmd.ocf from upstream

sample test run: http://ix64sap002.qa.suse.de/tests/432

### CODE REVIEW

File: tests/ha/clvm.pm [hunk 1]
```diff
@@ -29,15 +29,15 @@ sub run() {
         type_string "echo wait until clvmd resource is created\n";
     }
     else {
-        assert_script_run q(EDITOR="sed -ie '$ a primitive clvmd ocf:lvm2:clvmd op monitor interval=60 timeout=60'" crm configure edit);    #create clvmd primitive
-        assert_script_run q(EDITOR="sed -ie 's/^\\(group base-group.*\\)/\1 clvmd/'" crm configure edit);                                   #add clvmd to base-group
-        sleep 10;                                                                                                                           #wait to get clvmd running on all nodes
+        assert_script_run q(EDITOR="sed -ie '$ a primitive clvmd ocf:heartbeat:clvm op monitor interval=60 timeout=60'" crm configure edit);    #create clvmd primitive
```
Comment (sysrich) [inline tests/ha/clvm.pm]: Maintenance are likely to want to reuse our tests. is `ocf:heartbeat:clvm` available in SLE 12 GA and SP1? If no, then maybe this is a case of an `if` to check for the version and use the right clvmd RA for the right version.

Comment (vpelcak) [inline tests/ha/clvm.pm]: It does exist in both. 

"crm configure primitive clvm ocf:heartbeat:clvm" works.

However now it is simplified to "primitive clvm clvm".
That might be enough.

**Comment by okurz** [comment]
LGTM

**Comment by dzyuzin** [comment]
successful testrun: http://ix64sap002.qa.suse.de/tests/679 http://ix64sap002.qa.suse.de/tests/680

**Pull request merged**